lib/web_ui/lib/src/engine/compositor/fonts.dart

```diff
@@ -73,13 +73,35 @@ class SkiaFontCollection {
 
   js.JsObject getFont(String family, double size) {
     if (_registeredTypefaces[family] == null) {
-      if (family == 'sans-serif') {
-        // If it's the default font, return a default SkFont
-        return js.JsObject(canvasKit['SkFont'], <dynamic>[null, size]);
+      if (assertionsEnabled) {
+        html.window.console.warn('Using unregistered font: $family');
       }
-      throw Exception('Unregistered font: $family');
+      return js.JsObject(canvasKit['SkFont'], <dynamic>[null, size]);
     }
-    final js.JsObject skTypeface = _registeredTypefaces[family].values.first;
+    //final js.JsObject skTypeface = _registeredTypefaces[family].values.first;
+    // We don't attempt to find a Typeface matching the text style. Instead, we
+    // try to find the "default" typeface. The default typeface either has no
+    // descriptors, or only has a descriptor of font-weight 400 (the default).
+    final Map<Map<String, String>, js.JsObject> typefaces =
+        _registeredTypefaces[family];
+    js.JsObject skTypeface;
+
+    for (MapEntry<Map<String, String>, js.JsObject> entry
+        in typefaces.entries) {
```

Comment:
nit: `forEach` might be faster here because it doesn't need to allocate a `MapEntry` on each iteration.

Reply:
`forEach` doesn't allow you to terminate iteration early, keeping the for-loop for now